packages/babel-plugin-transform-typescript/test/realWorldTests.js

```diff
@@ -0,0 +1,338 @@
+/*
+Usage: `node ./test/realWorldTests`
+Does not run as part of normal testing (takes too long).
+
+This checks out a few repositories and tests the typescript transform on them.
+It checks that for every '.ts' file:
+
+- File parses correctly and produces a valid AST.
+- Generator outputs code which parses back to the same AST.
+- Transform produces output with an equivalent AST to the output produced by `tsc`. (Whitespace may differ.)
+*/
+
+const babel = require("babel-core");
```

Comment:
Looks like we'll want to update these all to `@babel/` and also make sure these are all called out in `devDependencies` otherwise we'll be using the versions that we used to self-host rather than the versions in the repository itself.